### DISCUSSION TITLE
Mastodon verification in HTML?

### DISCUSSION THREAD

**Discussion by andreacfromtheapp** [welpo/tabi#410] [Q&A] [answered]
Hi @welpo 

How can I inject ah HTML link into my home page to verify on Mastodon?

```html
<a rel="me" href="https://c.im/@andreacfromtheapp">Mastodon</a>
```

Please and thank you

**Comment by andreacfromtheapp** [comment, marked as answer]
never mind... i jut noticed "me" is already in rel!

**Comment by welpo** [comment]
Glad you figured it out! Since it's not obvious, I [added a mention](https://github.com/welpo/tabi/commit/5db70b7978e7141be9396d3f69375a87b2d44ed2) to this fact on the docs.

Thanks for using tabi~
